### PR TITLE
feat: error log on unsupported pg

### PR DESCRIPTION
**Pull request by chriswk**

### CODE REVIEW

File: src/lib/util/postgres-version-checker.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    config: IUnleashConfig,
+    settingStore: ISettingStore,
+): Promise<void> {
+    const logger = config.getLogger('server-impl/postgresVersionWarner');
+    const postgresVersion = await settingStore.postgresVersion();
+    const pgSemVer = semver.coerce(postgresVersion); // Postgres usually reports Major.Minor, semver needs a patch version included in string
+    if (pgSemVer !== null && lt(pgSemVer, MIN_SUPPORTED_POSTGRES_VERSION)) {
+        logger.error(
+            `You are using PostgreSQL version ${postgresVersion}. Unleash only supports Postgres 13 and newer. Unleash might work, but database errors will not be supported`,
+        );
```
Comment (chriswk) [inline src/lib/util/postgres-version-checker.ts:16]: I'd love some better text here. Suggestions?

Comment (gastonfournier) [reply src/lib/util/postgres-version-checker.ts:16]:
```suggestion
            `You are running an unsupported version of PostgreSQL: ${postgresVersion}. You'll have to upgrade to Postgres 13 or newer to continue getting our support.`,
        );
```

**Review by gastonfournier — APPROVED** [review]
Just a suggestion, take what you like drop what you don't

**Review by daveleek — APPROVED** [review]
Nice!

**Pull request merged**